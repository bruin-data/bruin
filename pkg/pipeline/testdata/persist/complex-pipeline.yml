name: complex-pipeline
schedule: hourly
start_date: "2024-01-01"
default_connections:
  gcp: prod-connection
  snowflake: snow-connection
retries: 3
concurrency: 5
catchup: true
agent: true
variables:
  env:
    type: string
    default: production
  debug:
    type: boolean
    default: false
tags:
  - production
  - critical
domains:
  - analytics
  - reporting
meta:
  owner: data-team
  cost-center: engineering