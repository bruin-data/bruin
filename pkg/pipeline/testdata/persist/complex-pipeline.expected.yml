name: complex-pipeline
tags:
  - production
  - critical
domains:
  - analytics
  - reporting
meta:
  cost-center: engineering
  owner: data-team
schedule: hourly
start_date: "2024-01-01"

default_connections:
  gcp: prod-connection
  snowflake: snow-connection
catchup: true
retries: 3
concurrency: 5
agent: true

variables:
  debug:
    default: false
    type: boolean
  env:
    default: production
    type: string
